### DISCUSSION TITLE
Is any way that when I re-set intial value dirty get false.

### DISCUSSION THREAD

**Discussion by nageshwari17** [react-hook-form/react-hook-form#7219] [Q&A]
Hi, Guys. Can I able set {shouldDirty: true} based on condition. As [https://codesandbox.io/s/ecstatic-frog-c857t?file=/src/App.tsx](https://codesandbox.io/s/ecstatic-frog-c857t?file=/src/App.tsx) this example . Intitialy I have firstName as 'test' , when I click set hello btn, field become dirty . and again I re-set intial value to field but field still dirty . Is any way that when I re-set intial value dirty get false.

**Comment by bluebill1049** [comment]
need to use : https://react-hook-form.com/api/useform/resetfield

**Reply by nageshwari17** [reply]
Hi @bluebill1049  Thank you for quick response . Is this resetField method available in v7.0.7 ? I am getting an error its not a function . 
Please check this [Example](https://codesandbox.io/s/ecstatic-frog-c857t?file=/src/App.tsx)
